osquery/tables/system/darwin/processes.cpp

```diff
@@ -266,6 +268,66 @@ void genProcNumThreads(QueryContext& context, int pid, Row& r) {
   }
 }
 
+uint64_t parseKCItem(kcdata_item_t item) {
+  if (item->type == STACKSHOT_KCTYPE_TASK_SNAPSHOT) {
+    struct task_snapshot_v2* tsv2 = (struct task_snapshot_v2*)item->data;
```

Comment:
Is `static_cast` applicable here instead of C-cast?

Reply:
Needs to be reinterpret but yeah, c is bad